src/sentry/rules/conditions/event_frequency.py

```diff
@@ -188,7 +188,9 @@ def get_rate(self, event: GroupEvent, interval: str, environment_id: str) -> int
         end = timezone.now()
         # For conditions with interval >= 1 hour we don't need to worry about read your writes
         # consistency. Disable it so that we can scale to more nodes.
-        option_override_cm = contextlib.nullcontext()
+        option_override_cm: contextlib.nullcontext | contextlib._GeneratorContextManager = (
```

Comment:
I think this should be `option_override_cm: AbstractContextManager[object] = ...`